sway-lib-std/src/ufp64.sw

```diff
@@ -0,0 +1,247 @@
+library ufp64;
+//! A wrapper around u64 type for a library for Sway for mathematical functions operating with signed 32.32-bit fixed point numbers. 
+
+use core::num::*;
+use ::assert::assert;
+use ::math::*;
+use ::logging::*;
```

Comment:
Why is logging here? It is not needed. This is tech debt that should be cleared up.